install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -40,19 +40,26 @@ spec:
       for: 10m
       labels:
         severity: critical
+    - alert: CannotRetrieveUpdates
+      annotations:
+        message: Cluster version operator has not retrieved updates since {{ "{{ $value | humanizeDuration }}" }}. Failure reason {{ "{{ $labels.reason }}" }}. Console {{ "{{ $labels.console_url.url }}" }}.
+      expr: |
+        (time()-cluster_version_operator_update_retrieval_timestamp_seconds{reason!="NoChannel"} >= 300) and on(foobar) (console_url > 0)
```

Comment:
Is this 5m delay for testing?  Are we bumping it to [an hour][1] or more before landing this PR?

[1]: https://github.com/openshift/cluster-version-operator/pull/357#discussion_r419699248

Reply:
Also, I think we should alert regardless of `console_url > 0`.  Not sure what the PromQL is for "group in the `console_url` if you can find it, but leave the label unset if you can't".  And once we have that, some sort of conditional in `message` to skip the console link if we couldn't find the URI?

Reply:
Yeah, all for testing. And just hacking around to try and get the console from that other metric.  Will re-do based on @simonpasquier suggestions.

Reply:
Changed delay back to an hour.
Inserted condition into annotation so only non-zero length console url message appears.